qemu/tests/cfg/device_assignment.cfg

```diff
@@ -7,7 +7,19 @@
     variants:
         - multi_pf_boot:
             type = boot
-            nics = "nic1 nic2"
+            nics = "nic1 nic2 nic3"
+            pci_assignable_nic1 = no
+            nic_model_nic1 = virtio
+            driver_option = 0
+            variants:
+                - 82576:
+                    pf_filter_re = 82576
+                - XL710:
+                    pf_filter_re = XL710
+                - 82599:
+                    pf_filter_re = 82599
+                - BCM57810:
+                    pf_filter_re = BCM57810
```

Comment:
@luckyh, also I doubt will this test be generic, for every card we need to add it here ?

Reply:
@balamuruhans @zhguo01 @yanan-fu @luckyh  I've moved the cards types to some other cfg. Please help review again, thanks!